library/ic-ledger-types/src/lib.rs

```diff
@@ -9,16 +9,18 @@
     clippy::missing_safety_doc
 )]
 
+use std::convert::TryFrom;
+use std::fmt::{self, Display, Formatter};
+use std::ops::{Add, AddAssign, Sub, SubAssign};
+
```

Comment:
Could we get rid of blank lines between imports?

Reply:
We use cargo fmt as the default code formatting tool for rust, cargo fmt is such a formatting style

Reply:
Of course, if the formatting style of the official team is different from cargo fmt, please let me know, and I will delete this blank line